src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/HttpApi/ApiClient.cs

```diff
@@ -534,12 +535,27 @@ private async Task<ResponseStreamHolder> CaptureMetricsAsync(int processId, int
             throw await CreateUnexpectedStatusCodeExceptionAsync(responseBox.Value).ConfigureAwait(false);
         }
 
-        public async Task<ResponseStreamHolder> CaptureExceptionsAsync(int processId, ExceptionFormat format, CancellationToken token)
+        public async Task<ResponseStreamHolder> CaptureExceptionsAsync(HttpMethod method, ExceptionsConfiguration configuration, int processId, ExceptionFormat format, TimeSpan timeout)
+        {
+            using CancellationTokenSource timeoutSource = new(timeout);
+
+            string json = JsonSerializer.Serialize(configuration, DefaultJsonSerializeOptions);
+
+            return await CaptureExceptionsAsync(
+                method,
+                new StringContent(json, Encoding.UTF8, ContentTypes.ApplicationJson),
+                processId,
+                format,
+                timeoutSource.Token);
+        }
+
+        public async Task<ResponseStreamHolder> CaptureExceptionsAsync(HttpMethod method, HttpContent content, int processId, ExceptionFormat format, CancellationToken token)
```

Comment:
This should be a private method and the original `CaptureExceptionsAsync(int processId, ExceptionFormat format, CancellationToken token)` should be added back that calls this method.